nextflow-lib/nextflow.config

```diff
@@ -111,12 +95,54 @@ profiles {
         aws {
             batch {
                 // NOTE: this setting is only required if the AWS CLI tool is installed in a custom AMI
-                cliPath = '/home/ec2-user/miniconda/bin/aws' // Need to (manually) create custom AMI first to access this folder 
+                cliPath = '/home/ec2-user/miniconda/bin/aws' // Need to (manually) create custom AMI first to access this folder
             }
             region = 'eu-west-2'
         }
         tower {
             workspaceId = '236739176715692'
         }
     }
+    googleBatch {
```

Comment:
Yes I have moved the properties over from GLS. The [docs](https://www.nextflow.io/docs/edge/google.html#configuration) do not have any specific recommendations for Batch

Reply:
The doc has some unique properties. Please see https://www.nextflow.io/docs/edge/config.html#config-google. They may or may not be useful for us.

In addition, Fusion file system is interesting (https://www.nextflow.io/docs/edge/google.html#fusion-file-system). I am not sure which version of the NXF we have. It requires 22.10.0 for AWS S3 and 23.02.0-edge for GCS. It can potentially make our implementations for AWS Batch and GCP batch more consistent and faster. I am not sure if there is performance gain or performance panelty for HPC-based implementations such as LSF and Slurm. Please take a look.

Reply:
I went through all the properties listed for Google after your suggestions.
I have added one more property: `google.batch.spot`.
For some reason the `google.lifeSciences.debug` doesn't seem to work. I will try it out again and submit a separate change if the outcome changes.